### PR TITLE
translations

### PR DESCRIPTION
**Pull request by janno42**
nothing exiting here - just translations :)

### CODE REVIEW

File: _1327/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -7,152 +7,947 @@ msgid ""
...
+msgid "You are only allowed to define a document"
+msgstr "Du kannst nur ein Dokument angeben"
+
+#: main/forms.py:26
+msgid "You must select a document"
+msgstr "Du musst ein Dokument auswählen"
+
+#: main/forms.py:45 main/forms.py:47
+msgid "This link is not valid."
+msgstr "DieDieser Link ist ungültig."
```
Comment (Nef10) [inline _1327/locale/de/LC_MESSAGES/django.po]: ?

File: _1327/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -7,152 +7,947 @@ msgid ""
...
+msgid "Cancel"
+msgstr "Abbrechen"
+
+#: main/views.py:89
+msgid "Successfully created menu item."
+msgstr "Menüeintrag erstellt."
+
+#: main/views.py:115
+msgid "Successfully edited menu item."
+msgstr "Menüeintrag erstellt."
```
Comment (Nef10) [inline _1327/locale/de/LC_MESSAGES/django.po]: bearbeitet?

File: _1327/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -7,152 +7,947 @@ msgid ""
...
+msgid "Date"
+msgstr "Datum"
+
+#: minutes/models.py:54
+msgid "State"
+msgstr "Status"
+
+#: minutes/models.py:55 minutes/templates/minutes_meta_information.html:6
+msgid "Moderator"
+msgstr "Moderator"
```
Comment (Nef10) [inline _1327/locale/de/LC_MESSAGES/django.po]: Gemäß unserer GO Sitzungsleitung

**Comment by janno42** [comment]
all fixed

File: _1327/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -7,152 +7,947 @@ msgid ""
...
+#: main/templates/abbreviation_explanation.html:69
+msgid "Add another abbreviation explanation"
+msgstr "Abkürzungserklärung hinzufügen"
+
+#: main/templates/menu_item_edit.html:9
+msgid "Edit menu item"
+msgstr "Menüeintrag bearbeiten"
+
+#: main/templates/menu_item_edit.html:11
+msgid "Create menu item"
```
Comment (Nef10) [inline _1327/locale/de/LC_MESSAGES/django.po]: Shouldn't the english be the same here?

File: _1327/locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -7,152 +7,947 @@ msgid ""
...
+#: main/templates/menu_items_index.html:13
+msgid "Main menu items"
+msgstr "Hauptmenü-Einträge"
+
+#: main/templates/menu_items_index.html:19
+msgid "Footer items"
+msgstr "Fußzeilen-Einträge"
+
+#: main/templates/menu_items_index.html:25
+msgid "Create new menu item"
```
Comment (Nef10) [inline _1327/locale/de/LC_MESSAGES/django.po]: Shouldn't the english be the same here?

File: _1327/locale/de/LC_MESSAGES/django.po (hunk 4 above)
Comment (janno42) [inline _1327/locale/de/LC_MESSAGES/django.po]: probably, but refactoring all ui strings will be done later.

**Pull request merged**